src/app/shell/details/competition-details/competition-details.component.ts

```
@@ -58,52 +73,85 @@ export class CompetitionDetailsComponent extends TabParamsComponent implements O
     size: PaginationConstants.WORKSHOPS_PER_PAGE
   };
 
+  public readonly ModalType = ModalConfirmationType;
```

Biome (2.1.2)

[error] Duplicate class member name "ModalType"



(lint/suspicious/noDuplicateClassMembers)